src/app/views/cheat-sheets/balancers/balancers.component.scss

```diff
@@ -1,4 +1,5 @@
 .balancer-icons>div {
     display: inline-block;
-    width: 50px
+    // Factorio item icons are 32x32, plus 3px padding on each side

```

Comment:
I wonder can we do margin:auto instead of using fixed widths?

Reply:
I tried a bunch of different CSS but wasn't able to get what I wanted (same appearance, not dependent on icon size), so I ended up just adding visibility:hidden to the inline style of the ones not included. I probably should have done this originally. https://github.com/deniszholob/factorio-cheat-sheet/pull/82/commits/1230d2e8e62653b9e197911acb2b9af8a74e5ea5